Source/WebCore/platform/image-decoders/bmp/BMPImageReader.cpp

```diff
@@ -462,10 +467,11 @@ bool BMPImageReader::processBitmasks()
 
 bool BMPImageReader::processColorTable()
 {
-    m_tableSizeInBytes = m_infoHeader.biClrUsed * (m_isOS21x ? 3 : 4);
```

Comment:
We can't remove this line, we're setting a data member here

Reply:
But I moved it to line 47 as:
`const size_t tableSizeInBytes = m_infoHeader.biClrUsed * (m_isOS21x ? 3 : 4);`

Reply:
Yes, that part is correct, but the class `BMPImageReader` has a property `size_t m_tableSizeInBytes`. If we remove this line, we will not be updating that property anymore. At that point this changes the semantics of the code.

In the future if someone adds a function that reads `m_tableSizeInBytes` expecting it to have been updated by `BMPImageReader::processColorTable()`, they will have problems.

Alternatively you can remove the `m_tableSizeInBytes` data member from the `BMPImageReader` class and keep the changes you have as-is, though someone more familiar with this class should comment on that.

Reply:
Ops! I got it. Pushed new [f6a20e4](https://github.com/WebKit/WebKit/commit/f6a20e490aaaacb07ef5f6d5bad425110e70e629).